Makefile

```diff
@@ -5,11 +5,12 @@ OPTIONS=-target=wasi -tags=purego
 default: all
 
 wit-deps-update:
+	rm -rf wit/deps
 	wit-deps update
 
 wit-bindgen:
 	rm -rf binding
-	wit-bindgen tiny-go --out-dir binding --rename-package binding --gofmt ./wit
+	wit-bindgen-go -v generate --world golem-go-bindings --out binding ./wit
```

Comment:
can we use `go tool witbindgen`, so install is handled by go? (requires go 1.24)

Reply:
it does not work for me

Reply:
```go: no such tool "wit-bindgen-go"```

Reply:
have you set up the project with `go get -tool`?

Reply:
https://tip.golang.org/doc/modules/managing-dependencies#tools

Reply:
it needs to be stored as a dep in go.mod

Reply:
> have you set up the project with go get -tool?

of course not. nothing suggested that anywhere :D